lib/internal/async_hooks.js

```diff
@@ -243,13 +245,84 @@ function restoreActiveHooks() {
   active_hooks.tmp_fields = null;
 }
 
+const triggerAsyncIdSymbol = Symbol('triggerAsyncId');
```

Comment:
nit: I think the guideline tells that symbols should be named like `const kTriggerAsyncId = Symbol('kTriggerAsyncId');`

Reply:
This is disappearing and being replaced by `trigger_async_id_symbol` which is used by `AsyncResource` and will become an internal symbol so I can share it with the native `PromiseWrap`.

Reply:
Either way, the symbol name and variable should ideally match for easier `grep`ping around the source tree?